app/src/main/java/com/cortado/minionify/mode/AppMode.java

```diff
@@ -0,0 +1,27 @@
+package com.cortado.minionify.mode;
+
+public enum AppMode {
+    MINION(1),
+    MANAGER(2);
+
+    private final int mValue;
+
+    AppMode(int value) {
+        mValue = value;
+    }
+
+    public int getValue() {
+        return mValue;
+    }
+
+    public static AppMode fromValue(int value) {
+        switch (value) {
```

Comment:
avoid `switches` when possible. use the `mValue` field and iterate the enum with `values()`

Reply:
Done.